js/screens/About/About.js

```diff
@@ -1,9 +1,38 @@
 import React from 'react';
 import { Text, View } from 'react-native';
-const About = () => {
+import Button from '../../components/Button';
+import styles from './styles';
+const About = ({ navigation }) => {
+  goAboutUs = () => {
+    navigation.navigate('AboutUs');
+  };
+  goOnBoarding = () => {
+    navigation.navigate('OnBoarding');
+  };
+  goCommunnityPartners = () => {
+    navigation.navigate('CommunityPartners');
+  };
+
   return (
-    <View>
-      <Text>About Screen</Text>
+    <View style={styles.container}>
+      <Button
+        onPress={goAboutUs}
```

Comment:
please do inline functions instead of creating separate functions.
<Button
        onPress={() => navigation.navigate('OnBoarding')}
        name="How it works"
        color="darkBlue"
        size="huge"
      />

let me know how it goes.